dir.props

```diff
@@ -100,6 +100,7 @@
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/nugetbuild/api/v3/index.json" />
+    <DnuSourceList Include="https:%2F%2Fwww.myget.org/F/roslyn-nightly/api/v3/index.json" />
```

Comment:
I'm not sure we want to add the roslyn-nightly feed. Can we just upload the dependent package to our buildtools or dotnet-core feed?